pkgs/applications/science/biology/obitools/obitools3.nix

```diff
@@ -13,6 +13,11 @@ pythonPackages.buildPythonApplication rec {
     sha256 = "1x7a0nrr9agg1pfgq8i1j8r1p6c0jpyxsv196ylix1dd2iivmas1";
   };
 
+  postPatch = lib.optionalString stdenv.isAarch64 ''
+      substituteInPlace setup.py \
+      --replace "-msse2," ""
```

Comment:
creating build/temp.linux-aarch64-3.9/build
creating build/temp.linux-aarch64-3.9/build/python
creating build/temp.linux-aarch64-3.9/build/python/obitools3
gcc -Wno-unused-result -Wsign-compare -DNDEBUG -g -fwrapv -O3 -Wall -O3 -w -I src -I src/libecoPCR -I src/libjson -fPIC -Isrc -Isrc/libecoPCR -Isrc/libjson -I/nix/store/2ywp6ybcal3ml1m8hf0k69m51347893k-python3-3.9.6/include/python3.9 -c build/python/obitools3/utils.c -o build/temp.linux-aarch64-3.9/build/python/obitools3/utils.o -msse2 -w -fPIC
gcc: error: unrecognized command line option ‘-msse2’
error: command '/nix/store/3qsj1r70nk9z3srav6rv8nldriy41vrp-gcc-wrapper-9.3.0/bin/gcc' failed with exit code 1
builder for '/nix/store/0fcsgzx14isjvwvbiywy2ipgw5rqid8z-obitools3-3.0.1b11.drv' failed with exit code 1
error: build of '/nix/store/0fcsgzx14isjvwvbiywy2ipgw5rqid8z-obitools3-3.0.1b11.drv' failed